### ISSUE TITLE
No Images Error

### ISSUE THREAD

**Issue by musharna** [mattocci27/LeafArea#6]
I'm using this package (specifically the dev version with parameter) for automated extraction of leaf trait data but recently ran into an error that I have been unable to solve. I've looked through the other threads and seen some troubleshooting that was recommended for a similar problem, so I tried those but it didn't help. I've used this package for months without issue but until recently it stopped working and I'm unsure why. Here is what I have tried thus far:

- Run from console or terminal (didn't work)
- update software (didn't work)
- ensure only jpg images (didn't work)
- tried jpg images I've previously measured with package (didn't work)
- tried short directory (didn't work)
- tried shortened command with only directory defaults (didn't work)
- reinstalled imageJ (didn't work)
- tried on backup laptop (DOES work, but don’t know why my main doesn’t?)

Here's an image of the error I get when I try to run it. 
![image](https://user-images.githubusercontent.com/96366172/197625032-266186c4-e90b-4d93-8a5a-88d434dd2cb5.png)

The error I am getting in the console is:
Error in `[.data.frame`(X[[i]], ...) : undefined columns selected

**Comment by mattocci27** [comment]
Hi, does the example code work?

```
ex.dir <- eximg()
run.ij(set.directory = ex.dir, save.image = TRUE)
```

Do you know if the version on imageJ (and Java) is same for your current laptop and your backup laptop?

**Comment by musharna** [comment]
Hi Masatoshi, 
Thanks for your quick reply! The example code does indeed work. I will say that my backup laptop previously didn't have imageJ installed so it has the default version, but I did have it installed prior on my current laptop, though I reinstalled it to be safe -- so they have the same version
As per the text on the bottom of imageJ (not sure if you were asking for another java version found somewhere else):
Both current and backup laptop have imageJ 1.53t; Java 1.8.0_345 [64-bit] 

The only other aspect between the two laptop that I can think off the top of my head as immediately different is my current laptop is on the windows insider (beta channel) while my backup doesn't have windows insider. 
If you think this might be pertinent these are the os builds they are on:
current: 22623.870 
backup: 22000.978

Let me know if there is any other troubleshooting I can attempt, I'm not sure but I feel like it may be an issue with java, however I really have no idea.

**Comment by mattocci27** [comment]
I'm sorry I have no ideas how to fix this. I also don't have a windows machine now and I can't reproduce the problem. Is it OK for you to use the other laptop for now?

**Comment by musharna** [comment]
Hi again,
I understand and I figured that may be a possibility. I planned to just use the backup for now to get the data in the interim. Thank you for your help, I appreciate your effort and your tool immensely either way. Let me know if you think of any possible troubleshooting steps I may have missed.

**Comment by musharna** [comment]
Hi again,
Very quick update:
I resolved the issue but was a bit distracted with the holidays until now to post my conclusion but essentially it goes back to your previous post regarding deep or long paths. While it is unclear while this issue began out of the blue essentially I found that it was having errors finding the image when the path was too long and if the image name itself was too long, though those may essentially be the same thing. If anybody else experiences similar issues I would recommend temporarily changing the images to be measured to a single character name and retrying again with a shorter path. That solved all my issues, though the errors still showed up on some images and not others.
